### DISCUSSION TITLE
Misterious option: Images copied by editor

### DISCUSSION THREAD

**Discussion by macmarrum** [freeplane/docs#127] [General]
`Preferences…->Behaviour->Rich-Text Editor->Images copied by editor`
Can anyone help me understand what the option does?

![Images-copied-by-editor](https://github.com/freeplane/docs/assets/7197131/b7a9ff65-c632-4a7b-8d4d-1af05217cf6c)

**Comment by dpolivaev** [comment]
If this option is set, the "rich text editor" used in note panel and in editor dialogs copies all images from sources given in <img> tags in files and substitutes them so that the resulting html can be used locally. This feature is implemented in shtml component.

**Reply by macmarrum** [reply]
I see.

I understand now that "images copied by editor" refers to the action of creating a copy of an image in a local directory named like the mind map with the suffix "_files".

Is this done only when pasting into SimplyHTML or are there any other cases when this functionality is applied?
